cli/azd/pkg/input/console.go

```diff
@@ -74,7 +74,7 @@ type Console interface {
 	// Use the returned io.Writer to produce the output within the previewer
 	ShowPreviewer(ctx context.Context, options *ShowPreviewerOptions) io.Writer
 	// Finalize the preview mode from console.
-	StopPreviewer(ctx context.Context)
+	StopPreviewer(ctx context.Context, keepLogs bool)
```

Comment:
Any thoughts about keeping the API simple by having `keepLogs` be simply part of `ShowPreviewerOptions` when the previewer is constructed?

The only downside is that you can't decide later on whether to keep logs or not. Maybe that is indeed the requirement here.

Reply:
The intention is to decide keeping or not the logs at the end. 

I considered adding a new bag of options for stop, like `StopPreviewerOptions`, but I will wait if there's ever some other setting that only applies to the stopping process